undead-discord.css

```diff
@@ -265,6 +284,63 @@
   position: unset !important;
 }
 
+/* Fixed help margin */
+.win-buttons {
+    display: none !important;
+}
+
+.header-toolbar {
+    margin-right: -152px;
+}
+
+/* Join guilds */
+.create-guild-container.deprecated .action.create .action-icon, .create-guild-container.deprecated .action.join .action-icon {
+    display: none !important;
+}
+
+.create-guild-container.deprecated .create-or-join .form-inner {
+    background: none;
+}
+
+.create-guild-container.deprecated {
+    height: 282px;
+}
+
+.create-guild-container.deprecated .action .btn {
+    margin-top: 10px;
+}
+
+.create-guild-container.deprecated .action .action-body {
+    color: var(--fg-color);
+}
+
+.create-guild-container.deprecated .action:hover.join .btn {
+    background-color: var(--bg-color) !important;
+}
+
+.create-guild-container.deprecated .create-or-join header {
+    margin-bottom: 32px;
+}
+
+/* Server settings */
+.popout-menu .popout-menu-item, .popout-menu .popout-menu-item:hover {
+    color: var(--fg-color);
+    font-weight: 500;
+}
+
+.status-picker .popout-menu-item .helper {
+    color: var(--secondary-fg-color);
+}
+
+.popout-menu .popout-menu-separator {
+    border-bottom: none;
+}
+
+/* Mentions */
+.protip.popout-footer .tip {
+    color: var(--fg-color);
+}
+
```

Comment:
This is a lot of crap. Same as earlier about moving it into the blocks. Not actually sure if it's worth adding this because it's so much code and effectively doesn't really do anything. Maybe send a separate PR for that too, so we can quickly merge the simple stuff.